### ISSUE TITLE
Unpack: snow

### ISSUE THREAD

**Issue by pbuttigieg** [Vocamp/Virtual-Hackahon-on-Glacier-topic#4]
@sjskhalsa and I noted that [snow ](https://docs.google.com/spreadsheets/d/1_Nm1vj2hM7a3Gewkqkrnkn0zSTGOQK5JH0tGj8PNIU8/edit#gid=32243948&range=30:30)differs from [rain](https://docs.google.com/spreadsheets/d/1_Nm1vj2hM7a3Gewkqkrnkn0zSTGOQK5JH0tGj8PNIU8/edit#gid=32243948&range=29:29) in that when we say "rain" we understand that we're talking about water droplets falling through the atmosphere. We refer to collected rain as rainwater or similar. 

Snow, in this regard, is ambiguous - it may be precipitating or it may be in the form of a mass of snow. 

Perhaps we need  a set of snow terms that's analogous to the rain case:
- mass of snow: for snow that has accumulated
- snowfall: for the precipitation process itself
- falling snow: snow that's a participant in a snowfall process

Feels a little strange, but may work....

**Comment by Garybc** [comment]
Yep, I think these type of distinctions are a way to go.  There is a process of rain as raining or snow snowing. There is the composition of rain as water in the form of droplets and there is the resultant aggregate that forms on the ground giving a snow field, puddle, lake etc.

**Comment by Garybc** [comment]
Closed by mistake.
